green-shop/source/sass/base/_global.scss

```diff
@@ -4,4 +4,90 @@
 body {
   font-size: $font-size-base;
   font-family: $font-family-base;
+  background: $body-color;
+}
+
+ul,
+li {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+p {
+  margin-bottom: 0;
+}
+
+%wrapper {
+  width: 100%;
+  max-width: 1140px;
+  margin: auto;
+  display: flex;
+
+  @include respond-to(desktop-and-down) {
+    padding: 0 15px;
+  }
+}
+
+.main {
+  @extend %wrapper;
+
+  flex-flow: column nowrap;
+}
+
+.btn:focus {
+  outline: none;
+  box-shadow: none;
+}
+
+.img-fluid {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  object-position: center;
+}
+
+.btn-green {
```

Comment:
Nếu là .btn-green  thì nó chỉ có tác dụng làm màu xanh thôi chứ nhỉ
Còn các thuộc tính khác mình tạo class btn.  Độ to của button có thể hiện là  .btn-xs btn-md btn-lg